source/max-staleness/tests/ReplicaSetNoPrimary/DefaultNoMaxStaleness.yml

```diff
@@ -8,14 +8,14 @@ topology_description:
     type: RSSecondary
     avg_rtt_ms: 50  # Too far.
     lastUpdateTime: 0
-    maxWireVersion: 6
+    maxWireVersion: 21
```

Comment:
21 is MongoDB 7.0. 7 is MongoDB 4.0. Is it worth changing these to the min server version supported? Or is is better to use 7.0?

Reply:
I bumped it way up to 21 so we don't have to do big PRs like this for quite a long time. Changing it to the current min server version supported would require us to do a PR like this every time we drop support for a server. If 3.6 EOL was 2021 and we're dropping driver support in 2024 then I'd assume we'd drop support for 7.0 in 2029.

Reply:
Makes sense to me.